src/vsr/replica.zig

```diff
@@ -5097,6 +4771,42 @@ pub fn ReplicaType(
                 },
             }
 
+            if (replica != self.replica) {
+                // Critical: Do not advertise a view/log_view before it is durable.
+                // See view_durable()/log_view_durable().
+                if (message.header.view > self.view_durable() and
+                    message.header.command != .request_start_view)
```

Comment:
`request_start_view` includes the view being requested. We don't need (or want) to persist that view until we have received the start-view in question.